model/CMakeLists.txt

```
@@ -281,6 +282,10 @@ foreach (xlen IN ITEMS 32 64)
                         --c-preserve rvfi_halt_exec_packet
                         --c-preserve print_instr_packet
                         --c-preserve print_rvfi_exec
+                        --c-preserve rvfi_write
+                        --c-preserve rvfi_read
+                        --c-preserve rvfi_mem_exception
+                        --c-preserve rvfi_wX
```

```suggestion
                        --c-preserve rvfi_wX
                        --c-preserve csr_name_map_forwards
```
Isn't this also needed?

---

Not sure why it worked without that, but added it just to be safe.